apps/ui/src/components/SpaceDelegates.vue

```diff
@@ -200,8 +291,20 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
         :open="delegateModalOpen"
         :space="space"
         :delegation="delegation"
+        :initial-state="delegateModalState"
         @close="delegateModalOpen = false"
       />
     </teleport>
   </template>
 </template>
+
+<style scoped>
+.clamped-text {
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  /* Fallback for browsers not supporting line clamp */
+  max-height: 44px;
+}
+</style>
```

Comment:
There is `line-clamp` in Tailwind that we can use (minus fallback, but I'm not sure how important it is for IE11, not sure how our page looks like in it already). We could even just do `line-clamp-2 max-h-[44px]` so we don't have to manage fallback styles ourselves.

Reply:
Did not know about this tailwind class